model/ftn/wmupdtmd.ftn

```diff
@@ -637,16 +639,30 @@
 ! 2.  Currents ------------------------------------------------------- /
 !
         CASE (2)
+!/SMC !!Li  For sea point current option FSWND.   JGLi08Feb2021
+!/SMC       IF( FSWND ) THEN
```

Comment:
Is the "sea point wind" addition separate to the SMC mulitgrid additions?
If so, we might want to add this in it's own PR.

Have we got a regression test for sea point wind/current inputs?

EDIT: I see that `SEAWND` actually already exists in develop - so ignore that; I had not realised sea point only winds had already been added.